src/_includes/site-warning.html

```diff
@@ -0,0 +1,3 @@
+<h3 class="site-warning">
+    !! Due to COVID19, The Computer Museum will only be taking visitors by <a href="contact">appointment</a> !!
+</h3>
```

Comment:
Two things here:
1. Consider making this either an `h1` -- if it's of supreme importance -- or a `p` -- if less so. But, as is, right now it results in skipped headings on all of your pages.
2. Your `href` needs to start with a `/` to identify that the link should be from the root of the site (or the domain name); you can see how this currently is broken on this page: https://chan-project4.glitch.me/gallery/ because, without the `/`, the `contact` gets added to `/gallery/` instead of starting from the root.
```suggestion
<h3 class="site-warning">
    !! Due to COVID19, The Computer Museum will only be taking visitors by <a href="/contact">appointment</a> !!
</h3>
```